tests/functional/regressions/test_issue188.py

```diff
@@ -39,6 +39,6 @@ def test_issue188_skipws_5():
         SPACE[noskipws]: ' ';
     ''')
     with pytest.raises(TextXSyntaxError,
-                       match='.*Expected \'bar \' at position'):
+                       match='.*\d+: Expected \'bar \''):
```

Comment:
seems a 'r' is missing: `r'.*\d+: Expected \'bar \''`

Reply:
Yup. good catch. Fixed.